template_src/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "my_app",
+  "version": "1.0.0",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "none specified",
+  "license": "none specified"
```

Comment:
Actually version by default will be set to 1.0.0. Users don't pass in a version. I'd suggest leaving the other variables in for users to edit manually. Name is the only user entered value we update automatically.